CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [2.4.0-beta.1] - 2021-10-29
```

Comment:
I don't think we should delete previous changes, even if pre-release.

Reply:
Done.